docs/mobile-apps/automated-testing/espresso-xcuitest.md

```diff
@@ -101,19 +102,19 @@ Both Test Object and Sauce Labs utilize CLI commands and YAML configuration file
 | Choose a tablet device only. | `--tabletOnly` | Must use YAML |
 | Ensure the device is connected to a cellular network. | Not supported | Must use YAML |
 | Choose any device where the name matches the regex. | `--deviceNameQuery` | Must use YAML |
-| Specify which test class to run. | `--testsToRun <class>` | Must use YAML |
+| Specify which test class to run. | `--testsToRun <class>` | `--testOptions.class` |
```

Comment:
That only works though if the user is using the CLI driven approach, which we have not yet documented. So you can't specify `saucectl run --testOptions.class`. Same goes for all the other framework specify flags.

Reply:
ok.. I'll remove these changes